Meta/lint-ports.py

```diff
@@ -347,6 +347,12 @@ def run():
     from_table_set = set(from_table.keys())
     ports_set = set(ports.keys())
 
+    # Dictionaries are sorted by the order of addition, converting to a set loses this property
+    if list(from_table.keys()) != sorted(from_table.keys(), key=str.lower):
```

Comment:
not sure why you''re mentioning sets here, there is no set in this

Reply:
It explains why I converted to .keys() manually, I can remove it though.